pillar/top.sls

```diff
@@ -0,0 +1,3 @@
+base:
```

Comment:
We won't use a `base` environment, instead have that one always-fail. Environments are MetalK8s-version specific.

Reply:
I'm not sure I understand, you mean we should use `metalk8s-2.0` instead of `base` ?

Reply:
Yes. We briefly discussed this in one of the meetings, but maybe it wasn't entirely clear. Should document this, I guess...

Reply:
No you right we want to use it with saltenv and pillarenv need to do some research about it